src/components/activity-completion/completion-page-content.scss

```diff
@@ -10,33 +10,45 @@
   .completion-text {
     color: $cc-teal-dark1;
     font-size: 22px;
+    font-weight: bold;
   }
 
   .progress-container {
     display: flex;
-    padding: 20px;
+    padding: 0.5em;
```

Comment:
Any reason to use em over px?  em is relative and is based on parent.  I use px since Michael's specs tend to be written that way.

Reply:
fixed